service/controller/v12/cloudconfig/master_template.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	// MasterCloudConfigVersion defines the version of k8scloudconfig in use.
 	// It is used in the main stack output and S3 object paths.
-	MasterCloudConfigVersion = "v_3_3_1"
+	MasterCloudConfigVersion = "v_3_3_2"
```

Comment:
We could combine the master and worker constants to only have one which we have to update.

Reply:
Good idea. I combined the 2 constants. For future updates we just vendor k8scc and update the constant.